atlp-rwanda/e-comm-team-techsmith-fn
# #185218917 Users should reset password

## What does this PR do?
- Users should be presented with a reset password link
## Description of Task to be completed?
- In case the users password is forgotten, functionality should exists to reset the password via an associated and email account.

- Users should be presented with a reset password link on the login page, there should also be another page for validating token sent via email and new password and confirm password should be inputted and validated.
## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout ft-Users-should-be-presented-with-a-reset-password-link-185218917`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`
## What are the relevant pivotal tracker/Trello stories?
 #185218917
## Screenshots (if appropriate)

## Questions:
![Screenshot (332)](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/123118676/1c94c96d-1d7f-4d9f-9889-c5278aed280d)

![Screenshot (335)](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/123118676/afbb87db-21ea-4f5d-9efc-be9b698fdaef)
